src/components/nav-bar/component.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Link from 'redux-first-router-link';
+import Dropdown from 'react-dropdown';
+
+import { menuItems } from './constants';
+
+import './styles.scss';
+
+const NavBar = ({ setRouter }) => {
+  const handleChange = (item) => setRouter({ type: item.type, pathname: item.path })
+
+  return (
+    <ul className="c-nav-bar">
+      {menuItems.map(
+        item => (!item.dropdown && !item.component)
+        ? <li className="nav-bar-item"><Link to={item.path}>{item.name}</Link></li>
+        : (item.dropdown && !item.component &&
+            <li className="nav-bar-item">
+                <Dropdown
```

Comment:
Do you think that adding a plugin is the correct way of implementing this? Not sure... the behavior is really strange right now....